<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>databaseViewForm</class>
 <widget class="QDialog" name="databaseViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="closeButtonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>540</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="campusTab">
    <attribute name="title">
     <string>Campuses</string>
    </attribute>
    <widget class="QTableView" name="campusesTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>781</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>450</y>
       <width>361</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="collegeSelectBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="displayDistButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Display Distances</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="souvenirTab">
    <attribute name="title">
     <string>Souvenirs</string>
    </attribute>
    <widget class="QTableView" name="souvenirTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>781</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>450</y>
       <width>361</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="collegeSelectBoxSouv"/>
      </item>
      <item>
       <widget class="QPushButton" name="displaySouvButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Display Souvenirs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="showCartButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>450</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Cart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addCartButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>450</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to Cart</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>closeButtonBox</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>databaseViewForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>databaseViewForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
